bHY6yh-de2332/cod-yyyiou_bHY6yh-de2332.xml

```diff
@@ -0,0 +1,170 @@
+                <lb ed="#0" break="no"/>linam
+                 domini ne abitias nec defitias cum ab
+                 <lb ed="#0"/> eo corripens <!--slight textual differences with words included in the bible that aren't in the text-->quem enim diligit dominus 
+                 <lb ed="#0"/>corripit et quasi pater in filio complacet 
+                 <lb ed="#0"/>sibi </quote>
+            </p>
+             <p xml:id="bHY6yh-de2332-d1e166">
+                <ref xml:id="bHY6yh-de2332-Rd1e169">¶Iesus filius sidrach in ecclesiastico capitulo XXXIIIo.</ref> <quote xml:id="bHY6yh-de2332-Qd1e170" source="http://scta.info/resource/sir33_1¶1-15">
+                <lb ed="#0"/>Timenti dominum non obtinet mala sed in
+                <lb ed="#0" break="no"/>temptatione
+                 deus illum conseruat et libe
+                 <lb ed="#0" break="no"/>rat
+                  a malis. </quote>
+            </p>
+            <p xml:id="bHY6yh-de2332-d1e178"> /
+       <ref xml:id="bHY6yh-de2332-Rd1e181">¶Idem capitulo XXVII</ref> <!--could not find a matching verse-->
+         <quote xml:id="bHY6yh-de2332-Qd1e180">homines
+         <lb ed="#0"/> iustos probat temptatio conbulationis. </quote>
+            </p>
+            <p xml:id="bHY6yh-de2332-d1e186"> 
+       <ref xml:id="bHY6yh-de2332-Rd1e191">¶Idem
+        <lb ed="#0"/> XXXI</ref> <quote xml:id="bHY6yh-de2332-Qd1e188" source="http://scta.info/resource/sir31_2¶6-10">Infirmitas grauis sobriam facit animam. </quote>
+            </p>
+            <p xml:id="bHY6yh-de2332-d1e194">
+                <ref xml:id="bHY6yh-de2332-Rd1e204"><lb ed="#0"/>¶Matthew in euangulo capitulo V</ref> <quote xml:id="bHY6yh-de2332-Qd1e198" source="http://scta.info/resource/mt5_10¶1-11"> beati qui persecti
+                <lb ed="#0" break="no"/>onem 
+                patiuntur propter iustitiam quoniam ipso
+                <lb ed="#0" break="no"/>rum 
+                est regnum celorum </quote>
+            </p> 
+            <p xml:id="bHY6yh-de2332-d1e204">
+                <ref xml:id="bHY6yh-de2332-Rd1e216">¶Jacobtis in epistola
+                <lb ed="#0"/> capitulo 1 </ref> <quote xml:id="bHY6yh-de2332-Qd1e208" source="http://scta.info/resource/iac1_12¶1-17">beatus uir qui suffert temptationem 
+                <lb ed="#0"/>qui cum probatus fuerit accipiet coronam 
+                <lb ed="#0"/>uite quam repromisit deus diligentibus 
+                <lb ed="#0"/>se </quote> <!--differences between text and bible but the verse seems to corroborate it but I am not completely postitve-->
+            </p>  
+            <p>
+                <ref xml:id="bHY6yh-de2332-Rd1e241">¶paulus ad romanos capitulo V</ref> <quote xml:id="bHY6yh-de2332-Qd1e231" source="http://scta.info/resource/rom5_3¶11-13 http://scta.info/resource/rom5_4¶1-4">Tribulatio patien
```

Comment:
Here's an example of the `@` replacement to pilcrows causes a problem, namely in the source attribute. But here the source attribute should really just be deleted. See the suggestion below.

```suggestion
            <p>
                <ref xml:id="bHY6yh-de2332-Rd1e241">¶paulus ad romanos capitulo V</ref> <quote xml:id="bHY6yh-de2332-Qd1e231">Tribulatio patien
```